content/blog/2021-02-18-dvc-2-0-pre-release.md

```diff
@@ -195,16 +195,19 @@ See all the runs:
 ```dvc
 $ dvc exp show --no-pager --no-timestamp \
         --include-params featurize.max_features,featurize.ngrams
-┏━━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
-┃ Experiment    ┃     auc ┃ featurize.max_features ┃ featurize.ngrams ┃
-┡━━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━┩
-│ workspace     │ 0.56359 │ 5000                   │ 3                │
-│ master        │  0.5674 │ 1500                   │ 2                │
-│ ├── exp-80655 │ 0.56359 │ 5000                   │ 3                │
-│ ├── exp-63ee0 │  0.5515 │ 5000                   │ 2                │
-│ ├── exp-9bf22 │ 0.56448 │ 4000                   │ 2                │
-│ └── exp-bb55c │ 0.57462 │ 3000                   │ 2                │
-└───────────────┴─────────┴────────────────────────┴──────────────────┘
+```
+
+```dvctable
+ ─────────────────────────────────────────────────────────────────────
+  **Experiment**          **auc**   **featurize.max_features**   **featurize.ngrams**
```

Comment:
The only thing I wasn't able to was color the cells, since I wasn't able to tell which columns were params and which were metrics 😅

Reply:
@iterative/docs, are we good to merge this pr or should we wait till we can get the header text colored?

Reply:
Colorizing already uncolored cols seems out of scope. Would be nice for docs pages though (no need to maintain blogs to that extent)...